ui/index.html

```diff
@@ -106,6 +106,7 @@
         <script src="js/models/option_collection.js"></script>
         <script src="js/models/user.js"></script>
         <script src="js/models/error.js"></script>
+        <script src="js/perf.js"></script>
         <!-- Controllers -->
```

Comment:
I always got "Error: [$injector:unpr] Unknown provider: PhSeriesProvider <- PhSeries <- PluginCtrl" error, anything I miss to include or import to use the PhSeries? @wlach